### DISCUSSION TITLE
Mention creating a Github personal access token when bootstrapping.

### DISCUSSION THREAD

**Discussion by SuperFluffy** [twpayne/chezmoi#1382] [Ideas]
I use GPG agent to provide SSH keys, but the configs to get that going smoothly are in my dotfiles.

Since github removed password auth on August 13, 2021, one now needs to create a personal access token and use it in place of the password. This [blog post] talks about the background, and [this guide] walks you through how to do that.

Maybe this can be added to the FAQ or even to the quick start guide? I was a bit stumped at first about what to do because I am not keeping up with github news so much. :)

[blog post]: https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/
[this guide]: https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token

**Comment by twpayne** [comment]
Thanks very much for opening this discussion!

The PR https://github.com/twpayne/chezmoi/pull/1385 adds an entry to chezmoi's how-to with the links you kindly supplied. Is this OK or can we make it better?
